packages/app/cypress/e2e/runner/sessions.ui.cy.ts

```diff
@@ -132,6 +132,29 @@ describe('runner/cypress sessions.ui.spec', {
     // cy.percySnapshot() // TODO: restore when Percy CSS is fixed. See https://github.com/cypress-io/cypress/issues/23435
   })
 
+  // https://github.com/cypress-io/cypress/issues/24208
+  it('does not save session when validation fails', () => {
+    loadSpec({
+      projectName: 'session-and-origin-e2e-specs',
+      filePath: 'session/new_session_and_fails_validation_retries.cy.js',
```

Comment:
Can we add and/or update this test to only fail validation 1 time so we can verify the second retry is 'created' a session instead of 'recreated' a session?

Reply:
Added in https://github.com/cypress-io/cypress/pull/24565/commits/731df94ce6655576392eca54c1833d7e653de079